pycryptobot.py

```diff
@@ -737,7 +737,8 @@ def executeJob(sc, market, granularity, tradingData=pd.DataFrame()):
             print ('   Buy Count :', buy_count)
             print ('  Sell Count :', sell_count, "\n")
 
-            print ('      Margin :', str(truncate((((sell_sum - buy_sum) / sell_sum) * 100), 2)) + '%', "\n")
+            margin_decimal = (sell_sum - buy_sum) / sell_sum if sell_sum else 0
+            print('      Margin :', str(truncate((margin_decimal * 100), 2)) + '%', "\n")
```

Comment:
`python3 pycryptobot.py --market ETH-USD --granularity 3600 --sim fast --verbose 1`
occasionally causes `DivisionByZeroError`

Reply:
Unless this is the cause of another bug? I was just assuming there were not sells or something

Reply:
Looking at this closer, it seems to be related to the verbosity...